src/app/pages/error/error.layout.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Flex, Text, color, Button } from '@stacks/ui';
+
+import { Title } from '@app/components/typography';
+import GenericError from '@assets/images/generic-error.png';
+
+interface ErrorProps {
+  body: string;
+  helpText: string[];
```

Comment:
This could just as well be `React.ReactNode[]`?

Reply:
Will change with refactor.